custom_event_checks/opslevel/service_infrastructure_check/aws-postgres-extended-support-check.yml

```
@@ -0,0 +1,34 @@
+---
+version: 1
+kind: generic
+spec:
+  name: AWS Postgres Extended Support Check
+  serviceSelector: ".[] | .name"
+  successCondition: '[.[] | select(.name == $ctx.alias) | .relatedResources.nodes[]
+    | select(.type == "Database") | select(.rawData.engine == "aurora-postgresql")
+    | (.rawData.engine_version | tonumber) >= 12 ] | all'
+  message: "{% if check.passed %}\n  ### Check passed\n  \U0001F389 Congratulations!
+    All postgres databases linked to this service are running version 12 or newer.\n{%
```

```suggestion
  message: |
    "{% if check.passed %}\n  ### Check passed\n  \U0001F389 Congratulations!
```
The `|` here ensures all the lines under `message:` are read in as is. Multiple lines separated by newlines just the way they appear here